sunpy/cm/cm.py

```diff
@@ -133,12 +142,13 @@ def get_cmap(name):
         raise ValueError("Colormap {name!s} is not recognized".format(name=name))
 
 
-def show_colormaps():
+def show_colormaps(mission=None):
     """Displays a plot of the custom color maps supported in SunPy.
 
     Parameters
     ----------
-    None : none
+    mission : string
+        Show only the color tables from a mission.
 
```

Comment:
now case insensitive search and renamed the parameter to `filter` to be more general.